### PR TITLE
[13.0][MIG] account_menu_invoice_refund

### PR DESCRIPTION
**Pull request by kongrattapong**
Ready to review.

### CODE REVIEW

File: account_menu_invoice_refund/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 Ecosoft Co., Ltd (https://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+{
+    'name': 'Accunt Menu - Invoice & Refund',
```
Comment (bodedra) [inline account_menu_invoice_refund/__manifest__.py]:
```suggestion
    'name': 'Account Menu - Invoice & Refund',
```

**Review by bodedra — COMMENTED** [review]
Minor one

File: account_menu_invoice_refund/__manifest__.py (hunk 1 above)
Comment (kongrattapong) [reply account_menu_invoice_refund/__manifest__.py]: changed.

**Comment by kittiu** [comment]
cc @kittiu

**Comment by kittiu** [comment]
Please make sure that the menu Invoice / Refund is under Refund menu (now it is under Receipt)

File: account_menu_invoice_refund/hooks.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    "out_receipt": "customer",
+    "in_invoice": "supplier",
+    "in_refund": "supplier",
+    "in_receipt": "supplier",
+}
+
+
+def post_load_hook():
+    @api.model
+    def new_default_get(self, fields):
```
Comment (kittiu) [inline account_menu_invoice_refund/hooks.py]: Now, we have 2 new_default_get(). Can change 2 different name.

**Review by kittiu — APPROVED** [review]

**Comment by kittiu** [comment]
@kongrattapong there is travis error, please check.
```
account_menu_invoice_refund/hooks.py:73:91: B950 line too long (90 > 80 characters)
```

**Comment by kongrattapong** [comment]
> @kongrattapong there is travis error, please check.
> 
> ```
> account_menu_invoice_refund/hooks.py:73:91: B950 line too long (90 > 80 characters)
> ```
@kittiu 
Thank for review.

**Comment by kittiu** [comment]
This module is ready for review/merge.

**Comment by kongrattapong** [comment]
@OCA/accounting-maintainers

**Comment by AaronHForgeFlow** [comment]
Please avoid merge commits when rebasing. Just ask for assistance in case you need help :)

**Comment by kittiu** [comment]
> Please avoid merge commits when rebasing. Just ask for assistance in case you need help :)

@AaronHForgeFlow sorry, but I am unsure what you mean :) any changes needed here?
Or may be I can redo this as new PR?

**Comment by kittiu** [comment]
There is design changed on the v12 
@kongrattapong may be it is better to re-migrate after https://github.com/OCA/account-invoicing/pull/707 is merged.

**Pull request closed without merging**